rest-api.md

```diff
@@ -0,0 +1,767 @@
+## API
+
+### /keys
+
+**GET** `/v1/keys`
+
+Endpoint returns list of keys for all modules.
+
+Query:
+
+- `used` - filter for used/unused keys. Possible values: true/false;
+- `operatorIndex` - filter for keys of operator with index `operatorIndex`;
+
+```typescript
```

Comment:
We don't send ts bytecode, but json.
Maybe use json schemas here instead of ts interfaces?
```json
{
  "object": "list",
  "data": [
    {
        "index": 1,
        "key": "string",
        "depositSignature": "string",
        "used": true,
        "operatorIndex": 1,
        "moduleAddress": "string",
    }
  ],
}
```

Reply:
Let's wait here for one more opinion; as for me, TypeScript interfaces are more readable.